tests/infra-validations.sh

```diff
@@ -78,41 +85,145 @@ echo "The remote_repo_url is $remote_repo_url"
 
 # Remove the user from the URL
 repo_url=$(getHostandPath "$remote_repo_url")
-
 git commit -m "inital commit"
 git tag "$tf_template_version"
 
 # git remote rm origin
-source=https://infra_account:$ACCESS_TOKEN_SECRET@$repo_url
+source=https://infra_account:$ACCESS_TOKEN_SECRET@$repo_url 
 git remote add origin "$source"
 echo "git push"
 git push -u origin --all
 git push origin "$tf_template_version"
 
-
 # Scaffold an Infra-HLD repo from TF Template ------------------
-cd ../..
+cd $TEST_WORKSPACE
+
 # Single Cluster scaffold for template
 pwd
 echo "../$terraform_template_dir"
 echo "$tf_template_version"
-spk infra scaffold -n $infra_hld_dir --source "$source" --version "$tf_template_version" --template "template" >> $TEST_WORKSPACE/log.txt
+mkdir $infra_hld_dir
+cd $infra_hld_dir
+
+spk infra scaffold -n $infra_hld_project --source "$source" --version "$tf_template_version" --template "template" >> $TEST_WORKSPACE/log.txt
```

Comment:
How can users view `spk infra scaffold` logs?

Reply:
Logs are published as artifacts in the smoke test pipeline. See: https://github.com/CatalystCode/spk/blob/master/smoke-test-pipeline.yml#L79